### DISCUSSION TITLE
What should we do when Redis is not available?

### DISCUSSION THREAD

**Discussion by cristipufu** [cristipufu/aspnetcore-redis-rate-limiting#16] [Polls]
Let's assume Redis is temporarily unavailable, what would you prefer?

**Comment by colbroth** [comment]
Is there a setting that would allow the request to go through or to fallback to in-memory rate limiting? currently I see an exception thrown when redis times out or is unavailable.

**Reply by cristipufu** [reply]
Unfortunately, not at this moment.

**Comment by yasir-fayrooz** [comment]
@cristipufu any update when this will be available? I see you updated the library to support .NET 10 but this is really needed especially for fault resilient microservices that rely on this library.
